cmd/latest.go

```diff
@@ -66,8 +66,8 @@ func latestPipeline(afs afero.Afero, skipPrereleases bool) (string, error) {
 	}
 
 	if removePrefix {
-		return fmt.Sprintln(strings.TrimPrefix(ver.Original(), "v")), nil
+		return fmt.Sprint(strings.TrimPrefix(ver.Original(), "v")), nil
```

Comment:
Hmm this does indeed seem to be adding an extra line, but without the `ln` the `fmt.Sprint` is now unnecessary 

```suggestion
		return strings.TrimPrefix(ver.Original(), "v"), nil
```